test/utils/runners.go

```diff
@@ -993,7 +993,6 @@ func MakePodSpec() v1.PodSpec {
 }
 
 func makeCreatePod(client clientset.Interface, namespace string, podTemplate *v1.Pod) error {
-	var err error
 	for attempt := 0; attempt < retries; attempt++ {
 		if _, err := client.Core().Pods(namespace).Create(podTemplate); err == nil {
```

Comment:
this actually looks like it was trying to save the error inside the loop for printing in the final return, but accidentally shadowing the err var